src/pyscaffoldext/django/extension.py

```diff
@@ -67,6 +77,17 @@ def enforce_django_options(struct, opts):
 
     return struct, opts
 
+def fix_django_settings(struct, opts):
+    old_string = f"'{opts['project']}"
```

Comment:
You are adding a single quote here? Is that the general idea? I wonder if a regex with either single or double quotes would be more robust in the case the Django project decides to start using double quotes (e.g. if they adopt black).

Reply:
you are right. I was little lazy, but If the replacement approach is ok for you, I make the fixes

Reply:
I did a quick read/search on how to use properly the "src" layout with django-manage, but I could not find anything (and I imagine you also did that).

I am under the impression that the replacement works because the new versions of python/setuptools support implicit namespacing. So in the end, `django-manage` wrongly considers `src` as a package... I wonder if messing around with `PYTHON_PATH` would solve the problem, but it is equally a not very clean solution.

In the end, if you have tested and it works even as a workaround, I don't have much to say.
If @FlorianWilhelm is fine with it, I would go forward after the fix in the tests and removing the f-strings. I don't see any more elegant way of doing that.